models/nc_news_models.js

```diff
@@ -20,7 +20,17 @@ const selectArticleByID = (id) => {
     })
 }
 
-const selectArticles = () => {
+const selectArticles = (query) => {
+    if (Object.keys(query).length > 0) {
```

Comment:
This outer check is redundant if you're then checking for the existence of a topic key on query.